README.md

```diff
@@ -301,4 +308,11 @@ Frequent Problems
   git it looks like you have deleted all files, when in fact they have
   never been checked out. Just do a checkout of the branch you want.
 
+* Receive `Error: repository has at least one unnamed head: hg r491`
```

Comment:
What do you mean with "Receive"? Why not just use the error message "`Error: repository has at least one unnamed head: hg rN`"  as the heading?

Reply:
For some reason I thought I saw this pattern elsewhere in the file. So I've removed that word "receive"